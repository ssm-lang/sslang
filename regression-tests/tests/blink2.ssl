main(led : & Int) -> () =
  loop
    after 50 , led <- 1
    wait led
    after 50 , led <- 0
    wait led
