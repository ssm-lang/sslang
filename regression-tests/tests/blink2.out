    50000000 led: 1
   100000000 led: 0
   150000000 led: 1
   200000000 led: 0
   250000000 led: 1
   300000000 led: 0
   350000000 led: 1
   400000000 led: 0
   450000000 led: 1
   500000000 led: 0
